pypeit/reduce.py

```diff
@@ -490,6 +497,8 @@ def run_extraction(self, global_sky, sobjs_obj, skymask, ra=None, dec=None, obst
             # Set to sciivar. Could create a model but what is the point?
             self.ivarmodel = np.copy(self.sciImg.ivar)
             # Set to the initial mask in case no objects were found
+            # TODO: fullmask is (or at least should be) an integer bit mask.  Is
+            # this assignment correct?
```

Comment:
I'm not sure I follow the question. Why would it not be?

Reply:
It's been a while, but I think I found somewhere where `fullmask` was being treated as a boolean mask.  I fixed it there, but I wanted to make sure that these `outmask` definitions were considered bitmasks and not boolean masks.  I've checked now and I think it's fine.  I've now altered these comments just to remind developers that these are bit masks.